mail/api.py

```diff
@@ -0,0 +1,66 @@
+"""
+Provides functions for sending and retrieving data about in-app email
+"""
+import requests
+
+from micromasters.settings import (
```

Comment:
`settings` isn't a package and we shouldn't import directly from `micromasters`. Instead this should be:

    from django.conf import settings

https://docs.djangoproject.com/en/1.10/topics/settings/#using-settings-in-python-code